CONTRIBUTING.md

```diff
@@ -137,11 +137,13 @@ Currently the process of releasing a new version of Langium is done manually one
     * Update version in `package.json`
     * Update dependency to `langium`
     * `npm install -g vsce`
-    * `vsce publish -p <token> --yarn` (the last argument is due to an npm bug)
+    * `vsce publish -p <token>`
     * `npm i -g ovsx`
-    * `ovsx publish -p <token> --yarn`
+    * `vsce package`
```

Comment:
We could change this procedure so we publish the same packaged file to both registries.